src/com/sap/piper/ConfigurationMerger.groovy

```diff
@@ -10,6 +10,8 @@ class ConfigurationMerger {
         Map filteredConfig = configKeys?configs.subMap(configKeys):configs
         Map merged = [:]
 
+        defaults = defaults ?: [:]
```

Comment:
Unfortunately not.

Reply:
The difference is: in case the parameter is ommited in the method signature the default value is used. In case <null> is explicitly used in the signature the <null> is used, not the default value ...